projects/plugins/protect/src/js/components/vulnerabilities-list/list.jsx

```diff
@@ -20,15 +20,20 @@ const VulAccordionItem = ( { id, name, version, title, icon, fixedIn, type } ) =
 				recordEvent( `jetpack_protect_${ type }_vulnerability_open` );
 			}, [ recordEvent, type ] ) }
 		>
-			<Text variant="title-small" mb={ 2 }>
-				{ __( 'How to fix it?', 'jetpack-protect' ) }
-			</Text>
-			<Text mb={ 2 }>
-				{
-					/* translators: Translates to Update to. %1$s: Name. %2$s: Fixed version */
-					sprintf( __( 'Update to %1$s %2$s', 'jetpack-protect' ), name, fixedIn )
-				}
-			</Text>
+			{ fixedIn && (
+				<Text variant="title-small" mb={ 2 }>
+					{ __( 'How to fix it?', 'jetpack-protect' ) }
+				</Text>
+			) }
+			{ fixedIn && (
```

Comment:
Very minor, is there a reason for having two separate `fixedIn` checks vs grouping the title and message in a single check? i.e.
```javascript
{ fixedIn && (
    <>
        <Text variant="title-small">...</Text>
        <Text mb={ 2 }>...</Text>
    </>
) }
```

Reply:
Nope, the IDE was complaining when I tried to group them and I didn't realize that I could just add an empty jsx element to be the parent of both and that would fix it. Thanks for the tip!